package.json

```diff
@@ -18,7 +18,6 @@
     "build": "tsc --build tsconfig.build.json",
     "build:clean": "npm run build -- --clean",
     "watch": "npm run build -- --watch",
-    "postinstall": "npx vscode-install",
```

Comment:
This command was installing VSCode specific TypeScript definitions and is replaced by `@types/vscode`